backend/apps/content/views.py

```
@@ -290,6 +290,18 @@ def get(self, request, pk):
 
         return response_obj
 
+class LessonAccessCheckView(APIView):
```

Ruff (0.15.20)

[error] Undefined name `APIView`

(F821)